test/sql-tap/insert3.test.lua

```diff
@@ -148,7 +148,7 @@ test:do_execsql_test(
             SELECT * FROM t2dup;
     ]], {
         -- <insert3-2.2>
-        4, 1, "b", "c", 5, -1, "987", "c", 6, -1, "b", "876"
+        4, 1, "", "", 5, -1, "987", "", 6, -1, "", "876"
```

Comment:
The reason of these changes worth to be described in the docs or in the commit message at least.

Reply:
Added to doc-request.